experiments/tsp/run-tsp.sh

```diff
@@ -12,14 +11,14 @@ clear
 source env.sh
 python run_tsp.py \
     --N=40 \
-    --use-gnn \
     --run=PPO \
+    --use-gnn \
     --lr=0.001 \
     --entropy-coeff=0.01 \
     --seed=0 \
+    --rollouts-per-worker=10 \
```

Comment:
not sure this changed anything

Reply:
so this should increase your sampling time by 10x from the default value of 1 -- but could lead to faster convergence if gradients are better estimated.  always a tricky one to tune, and of course for constant train_batch_size you'd have to scale it with number of workers